pkg/component/kubeapiserver/kube_apiserver_test.go

```diff
@@ -3785,7 +3785,6 @@ rules:
 
 			timer := time.AfterFunc(10*time.Millisecond, func() {
 				deploy.Generation = 24
-				deploy.Spec.Replicas = pointer.Int32(1)
```

Comment:
There is a nil pointer exception in the unit test, because `deploy.Spec.Replicas` is dereferenced 5 lines below.